uefi/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # uefi - [Unreleased]
 
+As of this release, the project has been relicensed from MPL-2.0 to
```

Comment:
nit: We could add "to align with the ecosystem" or so

Reply:
done